Northstar.CustomServers/mod/scripts/vscripts/sh_progression.nut

```diff
@@ -88,21 +88,22 @@ bool function ClientCommand_SetProgression( entity player, array<string> args )
 
 /// Resets a specific Titan's Aegis rank back to `0`
 /// * `player` - The player entity to perform the action on
-/// * `args` - The arguments passed from the client command. `args[0]` should be an integer corresponding to the index of the Titan to reset.
+/// * `args` - The arguments passed from the client command. `args[0]` should be a string corresponding to the chassis name of the Titan to reset.
```

Comment:
might be worth putting the vanilla ones here?

Reply:
Done.